build_manifest.yml

```diff
@@ -36,6 +36,7 @@ noir-packages-tests:
   rebuildPatterns: .rebuild_patterns_packages
   dependencies:
     - noir
+    - noir-packages
```

Comment:
My `noir-packages-tests` started failing tests after pulling master if I did not do this